alonzo/impl/src/Cardano/Ledger/Alonzo/Data.hs

```diff
@@ -150,8 +147,7 @@ instance (CC.Crypto c) => HeapWords (StrictMaybe (DataHash c)) where
 
 data AuxiliaryDataRaw era = AuxiliaryDataRaw
   { txMD' :: !(Map Word64 Metadatum),
-    scripts' :: !(StrictSeq (Core.Script era)),
-    dats' :: !(Set (Data era))
+    scripts' :: !(StrictSeq (Core.Script era))
```

Comment:
As discussed, we could drop this (the `AuxiliaryData` for Alonzo) entirely now, and use the Mary type.